From b96f7a0ee34387b36faa09710b1c303815eec6c2 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Mon, 5 Jun 2023 23:17:57 +0200
Subject: [PATCH] arm64: dts: st: add the dcmi instance in stm32mp251.dtsi

Add the node representing the dcmi IP in stm32mp251.dtsi

Change-Id: I19c1499d5ddc8a9f9528cb32782efa420e323470
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index af9001965b22..dbf5dbae98d1 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -1428,6 +1428,19 @@ uart8: serial@40380000 {
 				status = "disabled";
 			};
 
+			dcmi: dcmi@404a0000 {
+				compatible = "st,stm32-dcmi";
+				reg = <0x404a0000 0x400>;
+				interrupts = <GIC_SPI 141 IRQ_TYPE_LEVEL_HIGH>;
+				resets = <&rcc CCI_R>;
+				clocks = <&rcc CK_BUS_CCI>;
+				clock-names = "mclk";
+				dmas = <&hpdma 137 0x60 0x00003012>;
+				dma-names = "tx";
+				access-controllers = <&rifsc 88>;
+				status = "disabled";
+			};
+
 			adc_12: adc@404e0000 {
 				compatible = "st,stm32mp25-adc-core";
 				reg = <0x404e0000 0x400>;
-- 
2.39.5

